docs/ADOPTING_DARIA.md

```diff
@@ -0,0 +1,43 @@
+
+Read more about our workflow at: 
+https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/DCAF_101.md
+
+## Administration toolkit
+
+DARIA provides a number of workflows to administer and report on your funds activities. Analysts are able to export an anonymized spreadsheet for analysis; administrators are able to keep track of which clinics your fund  are and aren't working with; and your accounting team can use a specialized workflow to mark which pledges have been cashed.
+
+## Your DARIA instance and its data is yours
+
+Every fund's DARIA instance is separate from others. Your DARIA instance and its data is completely yours to use. DCAF and other DARIA funds do not have access to your data.
+
+## Security
+
+By reaching out to a fund, a patient is implicitly trusting us to be discreet and safeguard their personal data. To make good on funds' promise to keep our patients safe and offer them the best advice and support we can, our development team's top priority is keeping patient data secure and safe through deploying industry best practices and security monitoring softwares. 
+
+Additionally, we know any tech is only as secure as the people using it. A fund should also maintain secure data collection and usage practices at the individual case manager level, including deploying two factor authentication on their google accounts and using DARIA's features rather than using potentially compromising patient information in email communications.
+
+Find our complete security review at:
+https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/SECURITY.md
+
+## How to migrate
+
+Data migration is a manual process using a custom data entry form within DARIA. Funds should migrate data as close to the go-live date as possible. Plan to have a team of volunteers in your fund complete the migration. One person should be able to migrate approximately 200 patients over a weekend.
+
+We suggest breaking this down as follows:
+
+* Determine how much data to migrate (suggestion: current fiscal year or calendar year)
+* Recruit data entry volunteers, planning for 1 volunteer per 200 patients for 2 days of data entry
+* Break out data by volunteer to ensure no patient duplication
+
+## Costs
+
+DARIA costs start at $84 a year ($7 a month for 12 months), paid to DCAF once a year. This covers your share of server fees, and gets your fund its own URL and database. The DARIA engineering team will automatically update your instance with patches and new features. Funds which see a heavy amount of patient traffic (2,000+ new patients per year) may need to invest in a database upgrade at an additional $216 a year after the first year.
```

Comment:
I don't know if the below is appropriate / the level of detail you're looking for or if you don't want to mention the DDF relationship at all, but here's some additional details that could be added.

## Support and Maintenance

The DARIA engineering team will automatically update your instance with patches and new features. App monitoring is in place to alert the engineering team of any major outages. Urgent application support and troubleshooting is provided by the Digital Defense Fund by email at daria-admin@digitaldefensefund.org.

## How to move forward with DARIA

We are currently onboarding new funds at the rate of one fund per six months. To get the ball rolling getting your fund set up with DARIA, please reach out to daria-admin@digitaldefensefund.org.

Reply:
The support and maintenance tip is a great insight, though I'd strike the last sentence as that isn't in place yet, haha.

As noted above. I'd like to strike the moving forward thing for now, since this document is geared to a different goal than cold open or 'someone stumbles upon it'.